e2etest/test_manager.go

```diff
@@ -35,148 +31,50 @@ var (
 	babylonTag       = []byte{1, 2, 3, 4}                           //nolint:unused
 	babylonTagHex    = hex.EncodeToString(babylonTag)               //nolint:unused
 
-	// copy of the seed from btcd/integration/rpctest memWallet, this way we can
-	// import the same wallet in the btcd wallet
-	hdSeed = [chainhash.HashSize]byte{
-		0x79, 0xa6, 0x1a, 0xdb, 0xc6, 0xe5, 0xa2, 0xe1,
-		0x39, 0xd2, 0x71, 0x3a, 0x54, 0x6e, 0xc7, 0xc8,
-		0x75, 0x63, 0x2e, 0x75, 0xf1, 0xdf, 0x9c, 0x3f,
-		0xa6, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-	}
-
-	// current number of active test nodes. This is necessary to replicate btcd rpctest.Harness
-	// methods of generating keys i.e with each started btcd node we increment this number
-	// by 1, and then use hdSeed || numTestInstances as the seed for generating keys
-	numTestInstances = 0
-
-	existingWalletFile = "wallet.db"
-	exisitngWalletPass = "pass"
-	walletTimeout      = 86400
-
 	eventuallyWaitTimeOut = 40 * time.Second
 	eventuallyPollTime    = 1 * time.Second
+	regtestParams         = &chaincfg.RegressionNetParams
 )
 
-// keyToAddr maps the passed private to corresponding p2pkh address.
-func keyToAddr(key *btcec.PrivateKey, net *chaincfg.Params) (btcutil.Address, error) {
-	serializedKey := key.PubKey().SerializeCompressed()
-	pubKeyAddr, err := btcutil.NewAddressPubKey(serializedKey, net)
-	if err != nil {
-		return nil, err
-	}
-	return pubKeyAddr.AddressPubKeyHash(), nil
-}
-
 func defaultVigilanteConfig() *config.Config {
 	defaultConfig := config.DefaultConfig()
 	// Config setting necessary to connect btcd daemon
-	defaultConfig.BTC.NetParams = "simnet"
-	defaultConfig.BTC.Endpoint = "127.0.0.1:18556"
+	defaultConfig.BTC.NetParams = "regtest"
+	defaultConfig.BTC.Endpoint = "127.0.0.1:18443"
 	// Config setting necessary to connect btcwallet daemon
-	defaultConfig.BTC.BtcBackend = "btcd"
+	defaultConfig.BTC.BtcBackend = types.Bitcoind
 	defaultConfig.BTC.WalletEndpoint = "127.0.0.1:18554"
```

Comment:
One follow up task we can have is to remove any references to btcd backend, for example:
- `.WalletEndpoint` config option makes sense only in case of btcd/btcdwallet combo, in case of bitcoind there will be always one endpoint for bitcoind note/wallet
- `.BTC.BtcBackend ` options now makes little sense as we intend to support one backed ie bitcoind

Reply:
Makes sense to be a separate PR, as the config is used in other places also